src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx

```diff
@@ -264,7 +265,7 @@ const Index = props => {
                       onChange={handleChangeDynamicValue}
                       label='Nº Registros'
```

Comment:
label='Nº Registros'? That should be in internationalization, right?

Reply:
done